File: 백준/Bronze/11531. ACM 대회 채점/ACM 대회 채점.py
lst = [[0]*26 for _ in range(2)]
while 1:
    ipt = input().split()
    if ipt[0] == '-1':
        break
    m, nme, res = int(ipt[0]), ipt[1], ipt[2]
    if res == 'right':
        lst[1][ord(ipt[1])-65] = 1
        lst[0][ord(ipt[1])-65] += m
    else:
        lst[0][ord(ipt[1])-65] += 20
dap = 0
for i in range(26):
    if lst[1][i] == 1:
        dap += lst[0][i]
print(sum(lst[1]), dap)

File: 백준/Bronze/11531. ACM 대회 채점/ACM 대회 채점.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        int[][] lst = new int[2][26];
        Scanner sc = new Scanner(System.in);
        while (true) {
            String line = sc.nextLine();
            String[] ipt = line.split(" ");
            if (ipt[0].equals("-1")) {
                break;
            }
            int m = Integer.parseInt(ipt[0]);
            String nme = ipt[1];
            String res = ipt[2];
            int idx = nme.charAt(0) - 'A';
            if (res.equals("right")) {
                lst[1][idx] = 1;
                lst[0][idx] += m;
            } else {
                lst[0][idx] += 20;
            }
        }
        int dap = 0;
        int sum = 0;
        for (int i = 0; i < 26; i++) {
            if (lst[1][i] == 1) {
                dap += lst[0][i];
                sum++;
            }
        }
        System.out.println(sum + " " + dap);
        sc.close();
    }
}